canopen/objectdictionary/datatypes_struct.py

```diff
@@ -0,0 +1,62 @@
+import struct
```

Comment:
Wouldn't it be better named `datatypes_integer`?

Reply:
I was thinking if we get any future data types which are not limited to integer

Reply:
Well, there is no good reason for splitting out the module `datatypes_24bit` in the first place, other than it was the choice of the PR author.  We could just as well move the class definitions to `datatypes.py` directly.  They're small enough and don't introduce any new dependencies.

Reply:
I like that proposal. Let me do that.